CHANGES.md

```diff
@@ -1,3 +1,11 @@
+# grmtools 0.13.3 (2024-01-04)
+
+* Change generated code to avoid Clippy warnings about `unsafe`.
```

Comment:
> Change generated code to avoid Clippy warnings about `unsafe`.

It is actually a bit worse than that, because it is a rustc lint that was being turned into errors via `#[deny(rustc::unsafe_code)]`.
I would think something to the effect the following maybe?

* Change generated code to avoid errors about `unsafe` user code in yacc files.

Reply:
Good point! Fixed in 9d554dba84056354bab5314c2fec7df2a548c2ad.